examples/starwars/tests/websocket_utils.bal

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/io;
+import ballerina/websocket;
+
+const WS_INIT = "connection_init";
+const WS_ACK = "connection_ack";
+const WS_SUBSCRIBE = "subscribe";
+const WS_NEXT = "next";
+const WS_PING = "ping";
+const WS_PONG = "pong";
+
+isolated function validateConnectionAckMessage(websocket:Client wsClient) returns websocket:Error? {
+    json response = check wsClient->readMessage();
+    test:assertEquals(response.'type, WS_ACK);
+}
+
+isolated function sendConnectionInitMessage(websocket:Client wsClient) returns websocket:Error? {
+    check wsClient->writeMessage({'type: WS_INIT});
+}
+
+isolated function initiateGraphqlWsConnection(websocket:Client wsClient) returns websocket:Error? {
+    check sendConnectionInitMessage(wsClient);
+    check validateConnectionAckMessage(wsClient);
+}
+
+isolated function sendSubscriptionMessage(websocket:Client wsClient, string document, string id = "1",
+        json? variables = {}, string? operationName = ()) returns websocket:Error? {
```

Comment:
it seems the formatting is incorrect here.

Reply:
This is how the Ballerina formatter formats the code.